sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -81,12 +81,12 @@ def export(
     def force_flush(
         self,
         timeout_millis: float = 10_000,
-    ) -> None:
+    ) -> bool:
         """
         Ensure that export of any metrics currently received by the exporter
         are completed as soon as possible.
         """
-        # TODO
+        return True
```

Comment:
Is this just to confirm that the function completed correctly? There's no False scenario.

Reply:
Right, `flush` in metricexporters don't really do anything since the flushing mechanism is handled by the `BatchExportingMetricReader`, we just include them to be compliant with the spec.

See [this](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py#L158) as an example.